### ISSUE TITLE
Raspberry support of this.

### ISSUE THREAD

**Issue by RachitSharma123** [HackerHouseYT/AI-Smart-Mirror#91]
I just want to know that 
1. If I flashed MacOS on raspberry will this prog. work
2. will the 1 Gb Ram of raspberry will suffice the Running of multiple programs
3. are you still gonna make updates on this?

**Comment by viky12121200** [comment]
yes i m on woking on this and its working perfectly on rpi3 model b+

**Comment by enduser001** [comment]
have it working on tinkerboard with armbian. need to install a lot of the python dependencies.
